sdk/keyvault/azure-security-keyvault-keys/src/samples/java/com/azure/security/keyvault/keys/cryptography/CryptographyAsyncClientJavaDocCodeSnippets.java

```diff
@@ -69,8 +65,7 @@ public CryptographyAsyncClient createAsyncClient() {
      */
     public CryptographyAsyncClient createAsyncClientWithPipeline() {
         // BEGIN: com.azure.security.keyvault.keys.cryptography.async.cryptographyclient.pipeline.instantiation
-        RecordedData networkData = new RecordedData();
-        HttpPipeline pipeline = new HttpPipelineBuilder().policies(new RecordNetworkCallPolicy(networkData)).build();
+        HttpPipeline pipeline = new HttpPipelineBuilder().build();
```

Comment:
we could possibly introduce KVCredentialPolicy in the pipeline and pass in default azure credential there instead of the builder.
This will showcae better use of custom pipeline

Reply:
Discussed offline with @g2vinay . He has updates to the code snippets in his PR - #5701